### PR TITLE
change Game to Experience to match roblox ¯\_(ツ)_/¯

### PR DESCRIPTION
**Pull request by GalacticInspired**
sorry
not sorry

### CODE REVIEW

**Comment by LolloDev5123** [comment]
why do u not poo

**Comment by GalacticInspired** [comment]
you are poo

**Comment by LolloDev5123** [comment]
poo cannot speak

**Comment by LolloDev5123** [comment]
i am active inside this experience: "ara ara"

**Comment by GalacticInspired** [comment]
yes they can

**Comment by GalacticInspired** [comment]
we shouldn't be talking dirty on the intercom

**Comment by LolloDev5123** [comment]
bro parla italiano, così si ragiona

File: MainModule/Client/UI/Default/UserPanel.lua [hunk 1]
```diff
@@ -200,7 +200,7 @@ return function(data)
 				showItems()
 			else
 				items:Add("TextLabel", {
-					Text = "Cannot edit this table in-game";
+					Text = "Cannot edit this table in the experience";
```
Comment (p3tray) [inline MainModule/Client/UI/Default/UserPanel.lua:203]: Change
`Text = "Cannot edit this table in the experience";`
To
`Text = "Cannot edit this table while in an experience";`

**Review by p3tray — CHANGES_REQUESTED** [review]

File: MainModule/Client/UI/Default/UserPanel.lua (hunk 1 above)
Comment (GalacticInspired) [reply MainModule/Client/UI/Default/UserPanel.lua:203]: sure

**Comment by Sceleratis** [comment]
I genuinely hate this. There has to be a better word/phrase to refer to games/places that isn't "experience."

**Comment by Sceleratis** [comment]
"Cannot edit this table while in an experience" just sounds so clunky

**Comment by GalacticInspired** [comment]
sorry

I got a better solution

**Comment by LolloDev5123** [comment]
just stop

**Comment by ccuser44** [comment]
I disagree the naming of experience makes no sene. Keeping it as Game has no negative side effects while naming it to Experience does.

**Comment by Sceleratis** [comment]
Bippity Boppity Boop Denied.

**Comment by GalacticInspired** [comment]
But roblox isn’t a game lol

**Comment by Sceleratis** [comment]
> 
> But roblox isn’t a game lol

Correct, it is a game platform. Therefore, the things within it ("experiences") are games. 

"Roblox is an online game platform and game creation system developed by Roblox Corporation"

**Comment by Sceleratis** [comment]
That said they are trying to rebrand stuff on the platform as not just games.... but they're still just games....

**Comment by Expertcoderz** [comment]
> That said they are trying to rebrand stuff on the platform as not just games.... but they're still just games.... 

Precisely, and 'experience' sounds really awkward.

**Pull request closed without merging**